python/cudf/cudf/core/column/column.py

```diff
@@ -580,8 +580,8 @@ def _wrap_binop_normalization(self, other):
         if cudf.utils.utils.is_na_like(other):
             return cudf.Scalar(other, dtype=self.dtype)
         if isinstance(other, np.ndarray) and other.ndim == 0:
-            # Try and maintain the dtype
-            other = other.dtype.type(other.item())
+            # Return numpy scalar
+            other = other[()]
```

Comment:
I'm not that familiar with this notation, but I think this is equivalent. https://numpy.org/devdocs/reference/generated/numpy.ndarray.item.html
```suggestion
            other = other.item()
```

Reply:
`item` doesn't unfortunately work here since that can return a Python scalar. We want to return a scalar that holds numpy type information from the original array e.g. `1` that is `int8` or `uint16`

Reply:
Ah okay, thanks

Reply:
Do we need to be concerned if `other` is a numpy array with more than one element? From the [docs](https://numpy.org/doc/stable/user/basics.indexing.html#detailed-notes), a view is returned (not a scalar).

> An empty (tuple) index is a full scalar index into a zero-dimensional array. x[()] returns a scalar if x is zero-dimensional and a view otherwise. On the other hand, x[...] always returns a view.

Reply:
Oh cool, we're checking `ndim`, thanks